.ansible-lint

```diff
@@ -1,11 +1,4 @@
 ---
 skip_list:
-- '106'
-- '204'
-- '208'
 - '301'  # Commands should not change things if nothing needs doing
```

Comment:
are you planning to fix `301` issues?  This will likely be adding `changed_when: false` to tasks that use `command` and `shell` to get information from the system.